src/ucp/core/ucp_context.h

```diff
@@ -209,6 +209,19 @@ typedef struct ucp_am_handler {
     }
 
 
+#define UCP_CHECK_PARAM_NON_NULL(_param, _status, _action) \
+    if ((_param) == NULL) { \
+        ucs_error("The parameter %s must not be NULL", #_param); \
```

Comment:
should the error messages start with small letters? (same for all error msgs in this pr)

Reply:
yes